### ISSUE TITLE
libigl viewer isn't responding

### ISSUE THREAD

**Issue by dorsa01** [dilevin/computer-graphics-kinematics#60]
Hi

When I run kinematics.exe with any of the json files (I am using Windows), the libigl viewer doesn't respond and I get a blank screen. Any clue where the problem can be?

Thank you so much

**Comment by dexgs** [comment]
Is it possible that you have an infinite loop somewhere in your code? I think this would prevent the viewer from responding.

**Comment by elinsooon** [comment]
Hello, I'm having this same issue (also on Windows). All the code in main.cpp runs fine until v.launch() at the very end, so it would seem that all the files we worked on run correctly and without any completely obvious error

**Comment by dorsa01** [comment]
I actually tried to remove the loops to just check whether it works or not, but the problem persists. @elinsooon same happens for me.

**Comment by elinsooon** [comment]
Could it be an issue with the libigl install? If so what's the best way to go about fixing it

**Comment by lilygoli** [comment]
What do you see with just the blank given source files? If that gives you a viewer where you can navigate and rotate around the skeleton then probably it isn't a problem with your libigl installation.

**Comment by elinsooon** [comment]
What do you mean by blank given source files? As in just running the program with robot-arm.json?

**Comment by lilygoli** [comment]
yes and not filling in the files in src. would it give you a frozen window even with the empty files, as they are cloned and before filling in the files?
